odoo/addons/base/tests/test_ir_cron.py

```diff
@@ -32,6 +34,14 @@ def capture_triggers(self, cron_id=None):
 
 class TestIrCron(TransactionCase, CronMixinCase):
 
+    @classmethod
+    def setUpClass(cls):
```

Comment:
Because you have extra tests on this Class now. Can you move the `setUp` content in this new `setUpClass` (avoid recreating the same data for each test) ?